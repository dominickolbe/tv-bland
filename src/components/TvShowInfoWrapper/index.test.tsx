import { shallow } from "enzyme";
import React from "react";
import tvshowJSON from "../../__mock__/tvshow.json";
import { TvShowInfoWrapper } from "./";

describe("<TvShowInfoWrapper />", () => {
  it("renders snapshots", () => {
    const wrapper = shallow(<TvShowInfoWrapper tvshow={tvshowJSON} />);
    expect(wrapper).toMatchSnapshot();
  });
});
