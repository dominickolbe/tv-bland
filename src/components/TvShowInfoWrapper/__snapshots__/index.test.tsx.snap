// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TvShowInfoWrapper /> renders snapshots 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
  spacing={4}
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={3}
    sm={6}
    xs={6}
  >
    <WithStyles(ForwardRef(Typography))
      component="div"
      variant="h3"
    >
      <Styled(MuiBox)
        fontWeight="fontWeightMedium"
      >
        Genres:
      </Styled(MuiBox)>
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Typography))
      color="textSecondary"
      component="p"
      variant="body2"
    >
      Comedy
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={3}
    sm={6}
    xs={6}
  >
    <WithStyles(ForwardRef(Typography))
      component="div"
      variant="h3"
    >
      <Styled(MuiBox)
        fontWeight="fontWeightMedium"
      >
        Type:
      </Styled(MuiBox)>
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Typography))
      color="textSecondary"
      component="p"
      variant="body2"
    >
      Scripted
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={3}
    sm={6}
    xs={6}
  >
    <WithStyles(ForwardRef(Typography))
      component="div"
      variant="h3"
    >
      <Styled(MuiBox)
        fontWeight="fontWeightMedium"
      >
        Language:
      </Styled(MuiBox)>
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Typography))
      color="textSecondary"
      component="p"
      variant="body2"
    >
      English
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={3}
    sm={6}
    xs={6}
  >
    <WithStyles(ForwardRef(Typography))
      component="div"
      variant="h3"
    >
      <Styled(MuiBox)
        fontWeight="fontWeightMedium"
      >
        Status:
      </Styled(MuiBox)>
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Typography))
      color="textSecondary"
      component="p"
      variant="body2"
    >
      Ended
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
